packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/IfElsePreprocessorStatement.cs

```diff
@@ -18,12 +18,17 @@ public IfElsePreprocessorStatement(string condition, MethodBodyStatement ifState
         internal override void Write(CodeWriter writer)
         {
             writer.WriteLine($"#if {Condition}");
-            writer.AppendRaw("\t\t\t\t");
-            If.Write(writer);
+            using (writer.ScopeRaw(string.Empty, string.Empty, false))
```

Comment:
I added some comments showing we have the indentation wrong for this type of statement.